runtime/util/fltconv.c

```diff
@@ -424,6 +427,10 @@ fltconv_indexLeadingOne32(U_32 u32val)
 		leading = 7;
 		mask = 0x00000080;
 	}
+	// WARNING This loop will busy hang when compiled with
+	// gcc-linaro-4.9.4-2017.01 for armhf if the match should happen on
+	// the first iteration. The loop exit condition is moved to the end of
+	// the loop and doesn't execute before the first shift of mask.
```

Comment:
This comment still needs `/* */`

Reply:
Well, bother. Will fix. Thanks for catching.